docs/install/containerd/containerd-install.md

```diff
@@ -0,0 +1,99 @@
+# Install Kata Containers with containerd
+
+1. Perform initial checks
+
+   Check to see if Kata Containers is already installed:
+
+   ```bash
+   $ command -v kata-runtime &>/dev/null && { echo >&2 "ERROR: Please remove existing Kata Containers installation"; exit 1; }
+   ```
+
+1. Install dependencies
+
+   ```bash
+   $ source /etc/os-release || source /usr/lib/os-release
+   $ packages='containerd curl git jq'
+   $ case "$ID" in
+   $   ubuntu) sudo apt-get -y install $packages ;;
+   $   *) echo >&2 "ERROR: Unsupported distro: $ID"; exit 1 ;;
+   $ esac
+   ```
+
+1. Install Kata Containers
+
+   ```bash
+   $ tmpdir=$(mktemp -d)
+   $ pushd "$tmpdir" &>/dev/null
+   $ url="https://api.github.com/repos/kata-containers/runtime/releases/latest"
```

Comment:
The 2.0 releases are hosted here https://github.com/kata-containers/kata-containers/releases

Reply:
Since currently only pre-release version is available, the release url didn't work for me.  However I could verify rest of the instructions on Fedora 33
```
# kata-runtime --version
kata-runtime  : 2.0.0-rc0
   commit   : 84af1c20f34f9288bde422dc8bf7a0627b81c992
   OCI specs: 1.0.1-dev
# ctr run --runtime "io.containerd.kata.v2" --rm -t "$image" test-kata uname -m

x86_64
```

Reply:
Thanks for testing @bpradipt !

Reply:
@jcvenegas, @amshinde, @gnawux, @bergwolf - We have a bit of a problem here. Since [as @jcvenegas pointed out the `latest` URL is not guaranteed to list the latest release](https://github.com/kata-containers/kata-containers/pull/735#pullrequestreview-490811033), what do we do? For Kata 1.x, we rely on the distros package manager to determine the latest version, but here we can't.

Ironically, we *do* now have the facility to determine the latest release thanks to the `kata-check` changes on https://github.com/kata-containers/kata-containers/pull/735. But we then have a "chicken and egg" scenario - the user needs to know the latest version to download, but can't determine that without downloading a version including that change to `kata-check` :sob:

Maybe we could create a very small YAML file listing the latest releases for 1.x, 2.x and stable releases and host it on https://katacontainers.io (a bit like a cross between https://download.clearlinux.org/latest and the kernel version summary on https://www.kernel.org). That would require changes to our release tooling, `kata-check` and this document, but would solve the problem and would also isolate us from this odd GitHub behaviour.

Reply:
I think I've found a solution:

```bash
$ url="https://api.github.com/repos/kata-containers/kata-containers/releases"
$ latest_release=$(curl -sL "$url" | jq -r '.[].tag_name | select(contains("-") | not)' | sort -t "." -k1,1n -k2,2n -k3,3n | tail -1)
$ echo $latest_release
1.11.2
```

Reply:
Hmm. That's the result using the 2.x URL (which isn't quite right). But with 1.x:

```bash
$ url="https://api.github.com/repos/kata-containers/runtime/releases"
$ curl -sL "$url" | jq -r '.[].tag_name | select(contains("-") | not)' | sort -t "." -k1,1n -k2,2n -k3,3n | tail -1
1.11.3
```

Reply:
Eek - actually, that 1.11.2 *is* correct. tal at:

- https://github.com/kata-containers/runtime/releases/tag/1.11.3

  Looks fine.

- https://github.com/kata-containers/kata-containers/releases/tag/untagged-50bda11e5dc23f08b6ce

  Huh? That is the URL you get when you click "Kata Containers 1.11.3". It also hows as "draft" and is missing all assets!?!

/cc @egernst, @bergwolf.

Reply:
@jodh-intel I think we need to determine which release we want to download for 2.0 here. Right now there is no stable 2.0 release so all the releases are pre-release. Maybe we can just point users to `https://github.com/kata-containers/kata-containers/blob/2.0-dev/VERSION` to pick up the latest version. And once we tag the first 2.0 stable release, we can move to use `https://api.github.com/repos/kata-containers/kata-containers/releases/latest`.

In the meantime, we should stop tagging `1.x` releases on `kata-containers` repository. There is practically nothing for `1.x` to release here and we are using the runtime repository for the `1.x` releases.

WDYT?

Reply:
> Maybe we can just point users to https://github.com/kata-containers/kata-containers/blob/2.0-dev/VERSION

That's a good idea. But, assuming we merge `2.0-dev` into `master` when we release 2.0 (and I was assuming we'd do that), this idea would break the installation guide. However, why can't we merge `2.0-dev` into `master` "now" since this repo is purely for 2.x development? That way the doc can safely refer to https://github.com/kata-containers/kata-containers/blob/master/VERSION as that URL will then become "reliable".

> In the meantime, we should stop tagging 1.x releases on kata-containers repository.

Agreed. It's confusing to me (and maybe others) that we show 1.x releases in the 2.x repo.

/cc @amshinde, @egernst, @jcvenegas.

Reply:
Regarding the doc referring to https://github.com/kata-containers/kata-containers/blob/2.0-dev/VERSION, there *is* another solution to avoid breaking the doc - make the doc query the default branch for this repo. However, that's an extra step which arguably the install doc shouldn't need to do.

Reply:
We already have several places referencing `2.0-dev` branch of `kata-containers`. How about just using it for now and fixing all these places once we decide to move from `2.0-dev` to `master`.